### DISCUSSION TITLE
Can I connect with multiple channels - something like username/email/password, passwordless (with email) and discord connect ?

### DISCUSSION THREAD

**Discussion by 1swaraj** [Web3Auth/Web3Auth#329] [Help]
Is it possible to connect with username/email/password, passwordless (with email) and discord connect ?

**Comment by yashovardhan** [comment]
Yes it is possible to connect with multiple channels in Web3Auth. If you're a dApp integration Web3Auth and want to include multiple social channels to for your users customised for your UI, you have the use the `connectTo` function from the `web3auth/core` SDK for that: https://web3auth.io/docs/sdk/web/core/usage#logging-in-the-user

You can achieve the same with the `web3auth/web3auth` Modal SDK, and whitelabel it according to your needs: https://web3auth.io/docs/sdk/web/web3auth/


The only thing to note here is that you will have different [verifiers](https://web3auth.io/docs/custom-authentication/verifiers) for each of your different accounts. This means that you cannot link different social logins and the private key will be different for them. However, as a dApp you can get around it by asking us to create an Aggregate Verifier, which basically links multiple socials into a single verifier, helping users to link accounts.

**Comment by graphicmist** [comment]
> as a dApp you can get around it by asking us to create an Aggregate Verifier
Does this mean one have to ask you guys explicitly for creating Aggregate Verifier
